src/app/components/Menu/Menu.tsx

```diff
@@ -129,40 +156,59 @@ const Menu = () => {
                   {showLanguages && (
                     <div className="languagesListContainer">
                       <ul className="languagesList">
```

Comment:
is this list any different than the one below? you could extract this as a component instead of repeating it twice

Reply:
additionally the the logic is very redundant, is it not better to just not display the button of the language already set?